index.js

```diff
@@ -22,14 +21,22 @@ const exec = (cmd, args) => {
 	).filter(Boolean);
 };
 
+const install = (bin, useYarn) => () => execa(bin, ['install'])
```

Comment:
Awesome! I will look into it and add it here. _Sorry for the delays btw, been really busy_ :grimacing: